### DISCUSSION TITLE
Can I use on demand instances instead of spot?

### DISCUSSION THREAD

**Discussion by lenasappvia** [npalm/terraform-aws-gitlab-runner#1050] [Q&A]
Hello,

Thank you for a great module! I'd like to use on-demand instances instead of spot for the workload, how would I specify that?

Thank you!

**Comment by kayman-mk** [comment]
Sure you can and it is highly recommended for some tasks, e.g. `terraform apply`, ...

Make sure that you don't use `runner_worker_docker_machine_instance_spot` or `runner_instance.spot_price`. The module default is to use on-demand instances.

**Comment by lenasappvia** [comment]
Amazing, thank you so much for your fast reply, I appreciate it! :)
